ui/components/app/detected-token/detected-token.test.js

```diff
@@ -14,6 +14,13 @@ describe('DetectedToken', () => {
       metamask: {
         ...testData.metamask,
         ...mockNetworkState({ chainId: CHAIN_IDS.SEPOLIA }),
+        tokenBalances: {
+          '0x1': {
+            '0x514910771af9ca656af840dff83e8264ecf986ca': {
+              balance: '1000000000000000000',
+            },
+          },
+        },
```

Comment:
This also doesn't look like the right shape for tokenBalances state, same as https://github.com/MetaMask/metamask-extension/pull/28553/files#r1850903637